### PR TITLE
feature #1402 config cleanup - met_tool_wrapper

### LINKED ISSUE #1402: Clean up METplus use case configuration files
*Replace italics below with details for this issue.*

## Describe the Enhancement ##
*Provide a description of the enhancement request here.*

### Time Estimate ###
*Estimate the amount of work required here.*
*Issues should represent approximately 1 to 3 days of work.*

### Sub-Issues ###
Consider breaking the enhancement down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
*List relevant project deadlines here or state NONE.*

### Funding Source ###
2792541

### Assignee ###
- [ ] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [ ] Select **component(s)**
- [ ] Select **priority**
- [ ] Select **requestor(s)**

### Projects and Milestone ###
- [ ] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED PROJECT ASSIGNMENT** label
- [ ] Select **Milestone** as the next official version or **Future Versions**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [ ] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdatadb](https://github.com/dtcenter/METdatadb/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Enhancement Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Linked issues**
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by georgemccabe**
Another PR will follow for the model_applications conf files. I want to confirm that everything is good here before moving onto those use cases.

## Pull Request Testing ##

- [X] Describe testing already performed for these changes:</br>

Ran all use cases and diff tests

- [X] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

Review changes for consistency, copy/paste mistakes, etc.
Confirm format used for all conf files is acceptable
Ensure all automated tests pass

- [X] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [X] Do these changes include sufficient testing updates? **[Yes]**

- [X] Will this PR result in changes to the test suite? **[No]**</br>

- [X] Please complete this pull request review by **7/13/2022**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [X] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
- [X] Review the source issue metadata (required labels, projects, and milestone).
- [X] Complete the PR definition above.
- [X] Ensure the PR title matches the feature or bugfix branch name.
- [X] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [x] After submitting the PR, select **Linked issues** with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: parm/use_cases/met_tool_wrapper/GridStat/GridStat_comments.conf [hunk 1]
```diff
@@ -0,0 +1,250 @@
+# GridStat METplus Configuration
```
Comment (j-opatz) [inline parm/use_cases/met_tool_wrapper/GridStat/GridStat_comments.conf]: I'm curious about this use case template. Is this being preserved as a demonstration of the "old way" of detailing config files? Or is it going to be used in a different setting?

Comment (georgemccabe) [reply parm/use_cases/met_tool_wrapper/GridStat/GridStat_comments.conf]: This file was accidentally left in place. It was only created (renamed from the original) to easily show the proposed changes to solicit feedback. I will remove it now.

**Review by j-opatz — CHANGES_REQUESTED** [review]
It looks like a few of the automated testings failed: I'm not too concerned with the model application ones, since I believe that's related to a separate reorganization, but the met_tool_wrapper group 30-58 failed. It has some debug messages of "removing symbolic link", but I think the actual error is from the METdbLoad section. Can this be explained? If so, I have reviewed this PR and didn't find anything moved or printed in error so it's ready to be merged.

**Comment by georgemccabe** [comment]
@j-opatz, Yes, the failures in the tests are due to the s2s use cases using up all of the disk space. It appears to cause random other groups to fail because I have rerun them before and they ran fine. The met_tool_wrapper failure is due to the sql definition fine used to set up METviewer was moved from that repo into METdatadb (now METdataio). Pull request #1697 will resolve those issues.

**Pull request merged**